src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Color[] ALL_COLORS = Color.values();
+    private Random random = new Random();
```

Comment:
The `random` field should not be static since it's not a constant and its value is expected to change during the execution of the program. You should remove the `static` keyword from the `random` field declaration.